propagator/opentelemetry-propagator-ot-trace/README.rst

```diff
@@ -0,0 +1,48 @@
+OpenTelemetry OT Trace Propagator
+=================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-propagator-ot-trace.svg
+   :target: https://pypi.org/project/opentelemetry-propagator-ot-trace/
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-propagator-ot-trace
```

Comment:
Shouldn't this be called `opentelemetry-propagator-opentracing`?

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/302#discussion_r572337230